kiva/agg/freetype2/src/tools/docmaker/tohtml.py

```diff
@@ -1,9 +1,11 @@
 #  ToHTML (c) 2002, 2003, 2005, 2006, 2007, 2008
 #    David Turner <david@freetype.org>
 
-from sources import *
-from content import *
-from formatter import *
+import six
```

Comment:
`from __future__ import print_function` and you don't appear to be using `six` here